code/game/objects/items/ego_weapons/waw.dm

```diff
@@ -59,3 +59,61 @@
 	..()
 	combo += 1
 	force = initial(force)
+
+/obj/item/ego_weapon/totalitarianism
+	name = "totalitarianism"
+	desc = "When one is oppressed, sometimes they try to break free."
+	special = "This weapon attacks extremely slowly"
+	icon_state = "totalitarianism"
+	force = 120
+	damtype = RED_DAMAGE
+	armortype = RED_DAMAGE
+	attack_verb_continuous = list("cleaves", "cuts")
+	attack_verb_simple = list("cleaves", "cuts")
+	hitsound = 'sound/weapons/ego/hammer.ogg'
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 80
+							)
+
+/obj/item/ego_weapon/totalitarianism/melee_attack_chain(mob/user, atom/target, params)
+	..()
+	user.changeNext_move(CLICK_CD_MELEE * 3) // Slow
+
+
+/obj/item/ego_weapon/oppression
+	name = "oppression"
+	desc = "Even light forms of contraint can be considered totalitarianism"
+	special = "Use this weapon enter a stance to deflect bullets."
+	icon_state = "oppression"
+	force = 32
+	damtype = BLACK_DAMAGE
+	armortype = BLACK_DAMAGE
+	attack_verb_continuous = list("cleaves", "cuts")
+	attack_verb_simple = list("cleaves", "cuts")
+	hitsound = 'sound/weapons/ego/hammer.ogg'
+	attribute_requirements = list(
+							TEMPERANCE_ATTRIBUTE = 80
+							)
+	var/mode = 0
+
+/obj/item/ego_weapon/oppression/melee_attack_chain(mob/user, atom/target, params)
+	..()
+	user.changeNext_move(CLICK_CD_MELEE * 1.25) // FASTER
+
+/obj/item/ego_weapon/oppression/attackby(obj/item/P, mob/living/user, params)
```

Comment:
I believe you wanted to use attack_hand proc, when player uses it in hand.

Reply:
Yes sorry, I got then mixed up